shcDecoder/src/main/java/ca/bc/gov/shcdecoder/cache/ICacheManager.kt

```diff
@@ -0,0 +1,6 @@
+package ca.bc.gov.shcdecoder.cache
+
+interface ICacheManager {
+
+    suspend fun fetch()
```

Comment:
Think this is missing things, but I realise this is a draft

Reply:
This is all functional now check IMPL for the  implementation details.